doc/api/errors.md

```diff
@@ -2220,10 +2218,24 @@ A non-specific HTTP/2 error has occurred.
 Used in the `repl` in case the old history file is used and an error occurred
 while trying to read and parse it.
 
+<a id="ERR_INVALID_REPL_TYPE"></a>
+#### ERR_INVALID_REPL_TYPE
+
+> Stability: 1 - Experimental
+
+The `--entry-type=...` flag is not compatible with the Node.js REPL.
+
+<a id="ERR_INVALID_TYPE_FLAG"></a>
+#### ERR_INVALID_TYPE_FLAG
+
+> Stability: 1 - Experimental
+
+An invalid `--entry-type=...` flag value was provided.
```

Comment:
As these should directly exit the process without executing then entry, we probably don't need these errors, and should directly exit in the main script instead (this is what we normally do in the option parser).

Reply:
Removed, and lowered into the options parser.